bezier-rs/lib/src/lib.rs

```diff
@@ -540,10 +569,11 @@ impl Bezier {
 
 #[cfg(test)]
 mod tests {
-	use crate::utils;
-	use crate::Bezier;
 	use glam::DVec2;
 
+	use super::*;
+	use crate::utils;
```

Comment:
Please put this group above the `use glam::DVec2;` group since our ordering is *own crate* then *other first-party crates* then *std or third-party crates*.